test/Microsoft.Framework.Localization.Test/project.json

```diff
@@ -16,5 +11,8 @@
         "System.Reflection": "4.0.10-beta-*"
       }
     }
+  },
+    "commands": {
```

Comment:
Formatting here is missed up. BTW this is why these changes aren't usually worth it on their own - the overhead of tweaking things often ends up being a greater cost than the returned value.